src/api/xdsl/modem/xdsl-modem.v6.service.js

```diff
@@ -21,6 +21,11 @@ angular.module('ovh-api-services').service('OvhApiXdslModemV6', ($resource, OvhA
       method: 'POST',
       url: '/xdsl/:xdslId/modem/reconfigureVoip',
       interceptor,
+    },
+    exchangeModem: {
```

Comment:
Why not prefer name your action according to the endpoint API?

I mean `exchangeModem` > `comfortExchange`.

Reply:
I was not sure. So I do it ;-)